.github/workflows/keep-alive.yaml

```diff
@@ -0,0 +1,17 @@
+name: Keep
+on:
+  schedule:
+    - cron: "0 6 * * SUN"  # Once weekly on Sunday @ 0600 UTC 
+
+jobs:
+  keep-alive:
+    name: Alive
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gautamkrishnar/keepalive-workflow@v1
```

Comment:
Please use commit hashes instead of tags with workflows from untrusted sources. Otherwise, the source can inject any code into our organization. Dependabot can update hashes that match tags just fine ;)